projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -49,6 +56,7 @@ export class AddedToCartDialogComponent implements OnInit {
   constructor(
     protected modalService: ModalService,
     protected cartService: ActiveCartService,
+    @Inject(CartPromotionService)
     protected promotionService: PromotionService
```

Comment:
Since we can't configure injection via the cms coonfig, perhaps we directly use a `CartPromotionService` in the constructor?